### DISCUSSION TITLE
How to retrun to home screen after using home_button instead of app_drawer/search_screen ?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [MM2-0/Kvaesitso#685] [Q&A]
After launching app from search_screen/app_drawer, when i press home_button, the search_screen/app_drawer is still opened.

is there a way to return to blank home screen ?

**Comment by jack3308** [comment]
So, the work around I've used for the time being (only works with gesture navigation) monitors which app I'm in and based on that, the home gesture does different things.

If I'm already on the launcher then it just does the default "home" action so that the home gesture in KV settings still works.

If I'm in a different app, then it emulates the home button, waits 25 ms, then hits the back button so that I end up back on the home screen with next to no noticeable difference between just going home and this. It's fast enough that I had even forgotten about this fix until I saw your comment... 

I use XPosed edge for this (root) but you could probably use something like edge gestures combined with tasker to make it happen too if you don't have root.
